integration_test/cloudbuild-integration-test.yaml

```diff
@@ -47,13 +55,23 @@
   id: wait-for-adapter
   args:
   - ash
-  - integration_test/wait_for_port.sh
+  - integration_test/wait-for-port.sh
   - ${_ADAPTER_RUN_STEP}
   - ${_ADAPTER_PORT}
   waitFor:
-  - build-store-scu
+  - build-tools
   - setup-dataset-and-dicom-store
 
+- name: 'busybox:latest'
+  id: wait-for-storescp
+  args:
+  - ash
+  - integration_test/wait-for-port.sh
+  - ${_STORE_SCP_RUN_STEP}
+  - ${_STORE_SCP_PORT}
+  waitFor:
+  - build-tools
```

Comment:
Shouldn't this wait for store-scp instead?

Reply:
Same as above.